packages/web-app-user-management/src/views/Users.vue

```diff
@@ -0,0 +1,238 @@
+      :users="selectedUsers"
+      @cancel="toggleDeleteUserModal"
+      @confirm="deleteUsers"
+    />
+    <main class="oc-flex oc-flex-column oc-height-1-1 oc-p-m">
+      <app-loading-spinner v-if="loadResourcesTask.isRunning" />
+      <template v-else>
+        <div class="oc-app-bar">
+          <div class="oc-flex oc-flex-between">
+            <oc-breadcrumb class="oc-flex oc-flex-middle" :items="breadcrumbs" />
+          </div>
+          <div class="oc-flex-1 oc-flex oc-flex-start oc-mt-m">
+            <div v-if="selectedUsers.length" class="oc-flex oc-flex-middle">
+              <span v-text="selectedUsersText" />
+              <oc-button
+                id="files-clear-selection"
+                v-oc-tooltip="$gettext('Clear selection')"
+                :aria-label="$gettext('Clear selection')"
+                class="oc-ml-m"
+                appearance="outline"
+                @click="unselectAllUsers"
+              >
+                <oc-icon name="close" />
+              </oc-button>
+              <oc-button appearance="outline" class="oc-ml-m" @click="toggleDeleteUserModal">
+                <oc-icon name="delete-bin" />
+                <translate>Delete</translate>
+              </oc-button>
+            </div>
+            <div v-else>
+              <oc-button variation="primary" appearance="filled" @click="toggleCreateUserModal">
+                <oc-icon name="add" />
+                <translate>New user</translate>
+              </oc-button>
+            </div>
+          </div>
+        </div>
+        <no-content-message
+          v-if="!users.length"
+          id="user-management-users-empty"
+          class="files-empty"
+          icon="user"
+        >
+          <template #message>
+            <span v-translate>No users in here</span>
+          </template>
+        </no-content-message>
+        <div v-else>
+          <UsersList
+            :users="users"
+            :selected-users="selectedUsers"
+            class="oc-mt-m"
+            @toggleSelectUser="toggleSelectUser"
+            @toggleSelectAllUsers="toggleSelectAllUsers"
+          />
+        </div>
+      </template>
+    </main>
+  </div>
+</template>
+
+<script>
+import UsersList from '../components/Users/UsersList.vue'
+import CreateUserModal from '../components/Users/CreateUserModal.vue'
+import DeleteUserModal from '../components/Users/DeleteUserModal.vue'
+import AppLoadingSpinner from 'web-pkg/src/components/AppLoadingSpinner.vue'
+import NoContentMessage from 'web-pkg/src/components/NoContentMessage.vue'
+import { useStore } from 'web-pkg/src/composables'
+import { ref } from '@vue/composition-api'
+import { clientService } from 'web-pkg/src/services'
+import { useTask } from 'vue-concurrency'
+import { bus } from 'web-pkg/src/instance'
+import { mapActions } from 'vuex'
+
+export default {
+  components: { UsersList, AppLoadingSpinner, NoContentMessage, CreateUserModal, DeleteUserModal },
+  setup() {
```

Comment:
please do not mix composition api and options api

Reply:
? ^^

Reply:
@JanAckermann we discussed the options vs composition api topic today, we stick with the options api for now and do a workshop later this week to discuss the benefits of using the composition api. 

thanks for waiting ❤️